test/mlir/onnx/onnx_lowering.mlir

```diff
@@ -38,10 +38,10 @@ func @test_elementwise_op_with_scalar_values_2(%arg0 : tensor<f32>, %arg1 : tens
 
   // CHECK-LABEL: test_elementwise_op_with_scalar_values_2
   // CHECK: [[RES:%.+]] = alloc() : memref<f32>
-  // CHECK: [[LOAD1:%.+]] = load %arg0[] : memref<f32>
-  // CHECK: [[LOAD2:%.+]] = load %arg1[] : memref<f32>
+  // CHECK: [[LOAD1:%.+]] = affine.load %arg0[] : memref<f32>
```

Comment:
Is there a benefit in having affine loads/stores for zero dim memrefs? I do not see one, maybe I miss something?